10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,116 @@
+      <h3>Lorem ipsum.</h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia quo velit pariatur soluta quos voluptates officia nulla distinctio. Sunt, quas.
+      </p>
+    </article>
+    <article class="3box">
+      <img src="http://image.shutterstock.com/display_pic_with_logo/742138/109186895/stock-vector-a-illustration-based-on-aboriginal-style-of-dot-painting-depicting-being-and-seeming-109186895.jpg" alt="#" />
+      <h3>Lorem ipsum.</h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia quo velit pariatur soluta quos voluptates officia nulla distinctio. Sunt, quas.
+      </p>
+    </article>
+    <article class="3box">
+      <img src="http://image.shutterstock.com/display_pic_with_logo/742138/109186895/stock-vector-a-illustration-based-on-aboriginal-style-of-dot-painting-depicting-being-and-seeming-109186895.jpg" alt="#" />
+      <h3>Lorem ipsum.</h3>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia quo velit pariatur soluta quos voluptates officia nulla distinctio. Sunt, quas.
+      </p>
+    </article>
+  </section>
+  <section id="section4">
+    <article class="4box">
+      <img src="http://image.shutterstock.com/display_pic_with_logo/742138/109186895/stock-vector-a-illustration-based-on-aboriginal-style-of-dot-painting-depicting-being-and-seeming-109186895.jpg" alt="#" />
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Beatae, tempora!
+      </p>
+      <h4>08/16/16</h4>
+    </article>
+    <article class="4box">
+      <img src="http://image.shutterstock.com/display_pic_with_logo/742138/109186895/stock-vector-a-illustration-based-on-aboriginal-style-of-dot-painting-depicting-being-and-seeming-109186895.jpg" alt="#" />
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Beatae, tempora!
+      </p>
+      <h4>08/16/16</h4>
+    </article>
+  </section>
+</main>
+<footer id="foot">
+  <div class="topfoot">
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+    </p>
+  </div>
+  <div class="bottomfoot">
+    <button type="button" name="GO!">GO!</button>
+    <button type="button" name="GO!">GO!</button>
+  </div>
+</footer>
+
+  <!--   Including jQuery and the easing plug-in -->
+  <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>
+
 
+  <script>
+    $('#nav > ul > li > a, .business-name > a, footer >a ').bind('click', function() {
+      $('html, body').stop().animate({
+        scrollTop: $($(this).attr('href')).offset().top - 50
+      }, 1500, 'easeInOutExpo');
+      event.preventDefault();
+    });
+  </script>
 </body>
 </html>
```

Comment:
Good job with this assignment. The site itself looks fine and the markup and CSS is very readable and maintainable. Keep up the good work.